README.md

```diff
@@ -30,20 +74,26 @@ buildscript {
     }
   }
   dependencies {
-    classpath "gradle.plugin.org.jlleitschuh.gradle:ktlint-gradle:5.0.0"
+    classpath "gradle.plugin.org.jlleitschuh.gradle:ktlint-gradle:<current_version>"
   }
 }
 
 apply plugin: "org.jlleitschuh.gradle.ktlint"
 ```
 
+
+#### Using new plugin api
+
 Build script snippet for new, incubating, plugin mechanism introduced in Gradle 2.1:
 ```groovy
 plugins {
-  id "org.jlleitschuh.gradle.ktlint" version "5.0.0"
+  id "org.jlleitschuh.gradle.ktlint" version "<current_version>"
```

Comment:
Thoughts, it might be nice to try to use freshmark to automatically update our readme for us.
If you do choose to use it, just put a check around it's usage to disable if running on JDK 9+

https://github.com/diffplug/freshmark

Reply:
We already have an issue for such improvement: #77 
I consider to do it in a separate PR.